parse_date.re

```diff
@@ -1873,12 +1873,18 @@ timelib_time* timelib_strtotime(char *s, size_t len, timelib_error_container **e
 #define TIMELIB_CHECK_NUMBER                                           \
 		if (strchr("0123456789", *ptr) == NULL)                        \
 		{                                                              \
-			add_pbf_error(s, TIMELIB_ERR_UNEXPECTED_DATA, "Unexpected data found.", string, begin); \
+			char *msg = timelib_calloc(sizeof(char), 80 + strlen(format)); \
+			sprintf(msg, "Found character %c when expecting a number (0-9) in DateTime parsing format %s", *ptr, format); \
```

Comment:
DateTime is a PHP-ism, and timelib is used in other applications besides PHP as well. Can you re-word the message so it doesn't refer to PHP-related names? I would also put quotes (') around %c and %s.